src/app/containers/USElectionBanner/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { number, string, shape } from 'prop-types';
+import styled from 'styled-components';
+
+// Styling
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+} from '@bbc/gel-foundations/breakpoints';
+
+// Utilities
+import useToggle from '#hooks/useToggle';
+
+// Components
+import FrontPageSection from '../../components/FrontPageSection';
+
+const createMarkup = oembed => {
```

Comment:
create or `getMarkup`? 🤔

Reply:
I opted for create as is constructing the object required for `dangerouslySetInnerHTML` happy to change though.